docs/model_classes/usermodel.rst

```diff
@@ -201,11 +282,49 @@ It also shows one way of embedding models from a different system,
 in this case the
 `two-dimemensional polynomial model 
 <https://docs.astropy.org/en/stable/api/astropy.modeling.polynomial.Polynomial2D.html>`_
-from the AstroPy package.
+from the AstroPy package::
 
-.. literalinclude:: ../code/poly.py
+.. doctest-requires:: astropy
```

Comment:
This is being rendered in the text, so is the code actually being tested?

![image](https://github.com/sherpa/sherpa/assets/224638/04f74710-59e7-455c-9386-c1b89a2e5267)

Reply:
It didn't show up when I build locally and checked, but now I can see it locally, too. I must have changed something, but not sure what but I'll find it and fix.

Either way though, the test runs (if astropy is installed), which I verified by putting an error in the code and checked that it fails.

Reply:
OK, I think it stems from a missing black line higher up (something like "sphinx warning: bulleted list did not end on empty line"), which confused it. It's fixed in my local version, I'll check again after I push.